base/src/main/java/org/aya/concrete/desugar/BinExprParser.java

```diff
@@ -6,7 +6,7 @@
 import org.aya.concrete.Expr;
 import org.aya.concrete.error.OperatorError;
 import org.aya.generic.Constants;
-import org.aya.pretty.doc.Doc;
+import org.aya.prettier.doc.Doc;
 import org.aya.ref.DefVar;
 import org.aya.resolve.ResolveInfo;
```

Comment:
Please don't change `org.aya.pretty` in the pretty module.

Reply:
oh